docs/content/users/quickstart.md

```diff
@@ -397,6 +397,9 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
     ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
     ```
 
+!!!tip "Add Vite support?"
+    Since Laravel v9.19.0, Vite is included as the default [asset bundler](https://laravel.com/docs/11.x/vite). There are small tweaks needed in order to use it: [Working with Vite in DDEV - Laravel](https://ddev.com/blog/working-with-vite-in-ddev/#laravel).
```

Comment:
```suggestion
    Since Laravel v9.19.0, Vite is included as the default [asset bundler](https://laravel.com/docs/vite). There are small tweaks needed in order to use it: [Working with Vite in DDEV - Laravel](https://ddev.com/blog/working-with-vite-in-ddev/#laravel).
```

Let's use the Vite link without any version, otherwise we'll have to update it from time to time.

Reply:
done, thanks for feedback 👍